joomla/joomla-cms
# Handle non-existing manifest XML files in update sites

### Summary of Changes
If an extension is still in the system but the manifest file is gone a warning is thrown.


### Testing Instructions
1. Go to the filesystem folder `/administrator/manifests/packages/`
2. Rename the file `pkg_en-GB.xml` to `pkg_en-GB.xxml`
3. Login to Joomla backend
4. Click on System
5. Click on Global Configuration
6. Click on the Server Tab
7. Make sure Error Reporting is set to Development
8. Save & Close the changes
9. Click on System
10. Click on Update -> Update Sites
11. Notice the warning
12. Apply the patch
13. Reload the page
14. Warning is gone

### Expected result
No warning message

### Actual result
<img width="1161" alt="image" src="https://user-images.githubusercontent.com/359377/64470566-84a70e00-d145-11e9-966a-40de4a037a09.png">


### Documentation Changes Required
None
